pallets/cash/src/lib.rs

```diff
@@ -396,22 +388,23 @@ fn has_requisite_signatures(notice_state: NoticeState, validators: &Vec<Validato
     }
 }
 
+// XXX should this have all these printlns?
```

Comment:
They are actually very useful right now, which is why they have been kept in. This patch needs to make the corresponding change in types.json.